src/main/java/org/ieeervce/gatekeeper/Entity/User.java

```diff
@@ -0,0 +1,44 @@
+package org.ieeervce.gatekeeper.Entity;
+
+
+import jakarta.persistence.*;
+
+import java.time.LocalDateTime;
+
+@Entity
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long userId;
```

Comment:
Can we use `Long` instead?
Long can be set as null, so when we're creating a user we don't need to set this value

Reply:
Sure,will do that,but since I used the @GeneratedValue annotation,i thought that it by default would be generated without us needing to set.

Reply:
You're right
When creating a new object, the id is not to be populated. If we use `Long`, that will be represented as `null` instead of 0, which is more clear in meaning